ga4gh/cli.py

```diff
@@ -76,13 +76,16 @@ def server_main(parser=None):
 ##############################################################################
 
 
-class AbstractSearchRunner(object):
+class AbstractQueryRunner(object):
     """
-    Abstract base class for search runner classes
+    Abstract base class for runner classes
```

Comment:
Sure it is, we just don't have a compiler to enforce it ;)

In any case, the class name isn't really part of the PR